src/auditlog/appsco-audit-log.html

```diff
@@ -528,7 +528,8 @@
 //                            console.log('Property ' + property + ' change isn\'t supported in log.' );
                     }
 
-                    message += ' ' + this.localize(property) + ' ' + this.localize('account_log_profile_changed', 'fromValue', from, 'toValue', to);
+                    var propertyName = property.replace('_updated', '');
```

Comment:
Why you replaced '_updated' term? Why not just change localization message key in translations?